Makefile

```diff
@@ -48,13 +50,6 @@ build: bin
 flightctl-server-container:
 	podman build -f Containerfile -t flightctl-server:latest
 
-deploy-db:
-	cd deploy/podman && podman-compose up -d flightctl-db
```

Comment:
could we leave this as `deploy-db-podman` or allow override via ENV? As a dev I would like to iterate with the least amount of deps possible.

Reply:
It is in the deploy.mk file now, you can do deploy-db and it will only deploy the db. I moved all the deployment makefile targets to a separate file.

Is that what you are looking for?